src/harness/testcases/testdata/gwpz_record_0.json

```diff
@@ -0,0 +1,58 @@
+{"record":
+{
+  "id": "incumbent/wisp/0",
+  "type": "wisp",
+  "deploymentParam": 
+    {
+      "installationParam": {
+        "longitude": -99.7,
```

Comment:
You don't need this if you're injecting a GWPZ (this is only for GWBL, as someone reminded me in our call on Tuesday).